results/LDSDA.csv

```diff
@@ -0,0 +1,37 @@
+Point,x,y,Objective,Status,Time,Global_Time
+"[1, 1]",1,1,28,Optimal,1,1
+"[1, 2]",1,2,35,Optimal,1,2
+"[1, 3]",1,3,27,Optimal,1,3
+"[1, 4]",1,4,36,Optimal,1,4
+"[1, 5]",1,5,45,Optimal,1,5
+"[1, 6]",1,6,40,Optimal,1,6
+"[2, 1]",2,1,25,Optimal,1,7
+"[2, 2]",2,2,40,Optimal,1,8
+"[2, 3]",2,3,26,Optimal,1,9
+"[2, 4]",2,4,30,Optimal,1,10
+"[2, 5]",2,5,19,Optimal,1,11
+"[2, 6]",2,6,24,Optimal,1,12
+"[3, 1]",3,1,25,Optimal,1,13
+"[3, 2]",3,2,30,Optimal,1,14
+"[3, 3]",3,3,13,Optimal,1,15
+"[3, 4]",3,4,20,Optimal,1,16
+"[3, 5]",3,5,23,Optimal,1,17
+"[3, 6]",3,6,31,Optimal,1,18
+"[4, 1]",4,1,26,Optimal,1,19
+"[4, 2]",4,2,18,Optimal,1,20
+"[4, 3]",4,3,16,Optimal,1,21
+"[4, 4]",4,4,9,Optimal,1,22
+"[4, 5]",4,5,18,Optimal,1,23
+"[4, 6]",4,6,19,Optimal,1,24
+"[5, 1]",5,1,18,Optimal,1,25
+"[5, 2]",5,2,19,Optimal,1,26
+"[5, 3]",5,3,22,Optimal,1,27
+"[5, 4]",5,4,28,Optimal,1,28
+"[5, 5]",5,5,4,Optimal,1,29
+"[5, 6]",5,6,23,Optimal,1,30
+"[6, 1]",6,1,28,Optimal,1,31
+"[6, 2]",6,2,27,Optimal,1,32
+"[6, 3]",6,3,38,Optimal,1,33
+"[6, 4]",6,4,32,Optimal,1,34
+"[6, 5]",6,5,22,Optimal,1,35
+"[6, 6]",6,6,14,Optimal,1,36
```

Comment:
This is the LDSDA in a 6 times 6 lattice of Two External Variables Illustration.